framework-cloud/framework-git/src/main/java/org/kie/cloud/git/gogs/GogsGitProvider.java

```diff
@@ -0,0 +1,147 @@
+/*
+ * Copyright 2018 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+package org.kie.cloud.git.gogs;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.nio.charset.StandardCharsets;
+import java.util.Base64;
+import org.apache.http.HttpHeaders;
+import org.apache.http.HttpStatus;
+import org.apache.http.entity.ContentType;
+import org.kie.cloud.git.AbstractGitProvider;
+import org.apache.http.client.fluent.Request;
+import org.kie.cloud.git.constants.GitConstants;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class GogsGitProvider extends AbstractGitProvider {
+
+    private final String url;
+    private final String user;
+    private final String password;
+
+    private static final String URL_API_SUFFIX = "/api/v1/";
+    private static final String URL_CREATE_REPOSITORY_SUFFIX = "user/repos";
+    private static final String URL_REPOSITORY_SUFFIX = "repos/";
+    private static final String URL_GIT_SUFFIX = ".git";
+
+    private static final Logger logger = LoggerFactory.getLogger(GogsGitProvider.class);
```

Comment:
The same as before, LOGGER?

Reply:
We use logger with small letters everywhere in this test suite. I will keep it this way.

Reply:
Np!